docs/source/rbac.rst

```diff
@@ -1,14 +1,6 @@
 Role Based Access Control
 =========================
 
-.. note::
-
-  Prior to StackStorm 3.3, Extreme Networks provided a commercial version of the StackStorm automation platform which included Role Based Access Control (RBAC).
```

Comment:
Let's keep this note as it looks useful and also mention that RBAC is now available in StackStorm Open Source since version `3.4`.

Maybe something along the lines?
> Prior to StackStorm ``3.3``, Extreme Networks provided a commercial version of the StackStorm automation platform which included Role Based Access Control (RBAC). As these enterprise features were donated to the Linux Foundation, RBAC is now available in StackStorm Open Source since ``3.4``.

cc @m4dcoder WDYT?